### ISSUE TITLE
Finding tiff header file

### ISSUE THREAD

**Issue by manthey** [pearu/pylibtiff#66]
If you have a version of libtiff that doesn't have a pre-made python header file, and you are running python in a virtual environment, `tiff.h` is not located, since `sys.prefix` will be the virtual environment and not something like `/usr`.

On linux systems with gcc installed, perhaps some additional paths could be checked.  If the `tiff.h` header file has not been found, we could call `cpp -v` to get a list of the include paths that gcc would search, and then search each of those for `tiff.h`.  For instance, we could do:
```
    if not os.path.isfile(include_tiff_h):
        import subprocess
        try:
            cpp_info = subprocess.Popen(
                ["cpp", "-v", "-o", "/dev/null"], stderr=subprocess.PIPE,
                stdin=open('/dev/null', 'rb')).communicate()[1]
            cpp_paths = cpp_info.split("#include \"...\" search starts here:")[1].split("End of search list.")[0].split("\n")
            for cpp_path in cpp_paths:
                cpp_path = cpp_path.strip()
                if cpp_path.startswith('/'):
                    include_tiff_h = os.path.join(cpp_path, 'tiff.h')
                    if os.path.isfile(include_tiff_h):
                        break
        except Exception:
            pass
```
But there might be a better way than trying to parse the output of `cpp` for `#include "..." search starts here:` and `End of search list.`...

**Comment by manthey** [comment]
I can make a PR with the preceding code (or one with `tiff_h_4_0_8.py`, which happens to be identical to the existing `tiff_h_4_0_6.py`).

**Comment by pearu** [comment]
Resolved by PR #68
